packages/sqs/test/fakes/FakeConsumerErrorResolver.ts

```diff
@@ -1,15 +1,18 @@
 import { SqsConsumerErrorResolver } from '../../lib/errors/SqsConsumerErrorResolver'
 
 export class FakeConsumerErrorResolver extends SqsConsumerErrorResolver {
+  public errors: Error[]
   public handleErrorCallsCount: number
```

Comment:
🟢 I think that with `errors`,  `handleErrorCallsCount` is redundant

Reply:
fair point, we can just check the length